src/component/Multisearch/Multisearch.tsx

```diff
@@ -24,6 +24,7 @@ interface DefaultMultisearchProps {
 
 interface MultisearchProps extends Partial<DefaultMultisearchProps> {
   map: any;
+  wfsSearchBaseUrl?: string;
```

Comment:
the parameter is optional and has no default value anymore. What will happens if no `wfsSearchBaseUrl` will be passed to `WfsSearchInput` component?

Reply:
it will error of course :-) I cannot make that prop required, as the multisearch can be used without the wfs search. and i dont think there is a good default value?!

Reply:
i have added a console message if you use the wfs search without an url